code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -139,6 +146,8 @@
 	desc = "Used by engineering staff to see underfloor objects such as cables and pipes."
 	range = 2
 
+	darkness_view = 1
```

Comment:
🤨

Reply:
There's a ghost in my branch I swear bro